CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 5.8.0-BETA1 (2018-10-11)
+
+### Enhancements
+* Added new `ImportFlag` class that is used to specify additional behaviour when importing
```

Comment:
This could be incorporated in the next section and described slightly more on what it does instead of "additional behaviour" :-) What can a user do with this now?

Reply:
Otherwise fine.